doc/tutorials/starship.rst

```diff
@@ -37,46 +40,56 @@ To test connectivity, you can use the following command:
    You have successfully authenticated to the platform!
 
 
+.. _first_program:
+
 Executing your first program
 ============================
-The easiest way to execute a program using StarshipEngine is to create a blackbird script (an ``xbb`` file) and place it in your current working directory. Check the :ref:`blackbird tutorial <tutorial>` for how to create this file.
 
-For this example, we will use the following file and save it to ``test.xbb`` in our current working directory:
+The easiest way to execute a program using StarshipEngine is to create a Blackbird script (an ``xbb`` file)
+and place it in your current working directory. Check the :ref:`Blackbird tutorial <blackbird>` for how to create this file.
+
+For this example, consider the following Blackbird script, which represents a quantum program that matches
+exactly the gate layout of the Chip0 photonic hardware device. We will save the following file as ``test.xbb``
+in our current working directory:
 
 .. code-block:: python
 
-   name template_2x2_chip0
-   version 1.0
-   target chip0 (shots = 50)
+   name template_2x2_chip0     # Name of the program
+   version 1.0                 # Blackbird version number
```

Comment:
Yep - this is the version of the Blackbird language, i.e., the overall grammar, syntax, and intrinsic functions etc. If there are any changes to Blackbird as a language, we will increment the version number here. This will allow the parser to support multiple versions of the Blackbird language.